mozilla/foundation.mozilla.org
# removed all instances of ai_uses_personal_data

Closes #7693 


**Link to sample test page**: 
CMS: https://foundation-s-remove-ai--o5oogr.herokuapp.com/cms/pages/23/edit/
Front-end: https://foundation-s-remove-ai--o5oogr.mofostaging.net/en/privacynotincluded/eat-when-lose-its-instead-unit-fine/

**Steps to test**: 

1. If you would like to verify the field is removed from the CMS, please select the first link above, and scroll down to the AI section.
2. To test the front end, please visit the 2nd link above and scroll down in the page to verify that the "AI uses personal data" field has been removed.
3. If everything is looking/behaving as expected, testing is complete! 


## Checklist


**Changes in Models:**
